.mergify.yml

```diff
@@ -0,0 +1,23 @@
+queue_rules:
+  # This rule is for https://docs.mergify.com/commands/queue/
+  # and can be triggered with: @mergifyio queue
+  - name: default
+    merge_conditions:
+      # all github action checks in this list are required to merge a pull request
+      - check-success=Attributes
+      - check-success=Check
+      - check-success=Outpaths (aarch64-darwin)
+      - check-success=Outpaths (aarch64-linux)
+      - check-success=Outpaths (x86_64-darwin)
+      - check-success=Outpaths (x86_64-linux)
+      - check-success=Process
+      - check-success=Request
+      - check-success=Tag
+      - check-success=editorconfig-check
+      - check-success=label-pr
+      - check-success=nix-files-parseable-check
+      - check-success=nixfmt-check
+      - check-success=nixpkgs-vet
+    merge_method: rebase
+    # queue up to 5 pull requests at a time
+    batch_size: 5
```

Comment:
I suspect this will be used a lot, 5 PRs in queue is just not enough.

Reply:
Do you think so? There is a merged pull request every 10 minutes maybe, this would hardly fill up 5 PRs into one batch.

Reply:
Well as commiters most of us just wait for a PR to complete the CI process and then merge. With mergify this process can speed up a lot potentially, multiple reviewers adding multiple PRs to the queue. 


Anyways, I guess we could give it a shot as an initial value, raise it if need arises later.

Reply:
Well we can always increase it. This is just a trade off so that if one pull request fails the merge queue doesn't have to split things up. Just now it seems that on average  (using the weekly statistics) we would have one pull request in the pipeline.